.ci/pitest-survival-check-xml.groovy

```diff
@@ -313,70 +311,122 @@ private static void printMutation(String flag, Mutation mutation) {
 private static Set<Mutation> setDifference(final Set<Mutation> setOne,
                                            final Set<Mutation> setTwo) {
     final Set<Mutation> result = new HashSet<>(setOne)
-    result.removeIf(mutation -> setTwo.contains(mutation))
+    result.removeIf { mutation -> setTwo.contains(mutation) }
     return result
 }
 
 /**
  * A class to represent the XML {@code mutation} node.
  */
-@EqualsAndHashCode(excludes = ["lineNumber", "unstable"])
-record Mutation(String sourceFile, String mutatedClass,
-                String mutatedMethod, String mutator,
-                String description, String lineContent, int lineNumber,
-                boolean unstable)
-        implements Comparable<Mutation> {
+@EqualsAndHashCode(includeFields = true, excludes = ["lineNumber", "unstable"])
+final class Mutation implements Comparable<Mutation> {
 
     /**
      * Mutation nodes present in suppressions file do not have a {@code lineNumber}.
      * The {@code lineNumber} is set to {@code -1} for such mutations.
      */
     private static final int LINE_NUMBER_NOT_PRESENT_VALUE = -1
 
+    private final String sourceFile
+    private final String mutatedClass
+    private final String mutatedMethod
+    private final String mutator
+    private final String description
+    private final String lineContent
+    private final int lineNumber
+    private final boolean unstable
```

Comment:
If we remove "private", will Groovy generate getters for us?

Reply:
Yes, it will. Done. https://groovy-lang.org/style-guide.html#_getters_and_setters